src/pages/graphile-build/hooks.md

```diff
@@ -115,7 +98,7 @@ used ones:
 - `scope` - an object based on the third argument to `newWithHooks` or
   `fieldWithHooks`; for deeper hooks (such as `GraphQLObjectType:fields:field`)
   the scope from shallower hooks (such as `GraphQLObjectType`) are merged in.
-- `Self` - only available on hooks that are called after the object is created
+- `Self` - only available on deferred hooks that are called after the object is created
```

Comment:
This sentence is now a little redundant, lets move the redundancy to parenthesis:

```suggestion
- `Self` - only available on deferred hooks (those that are called after the object is created,
   e.g. `GraphQLOb...
```